setup.cfg

```diff
@@ -293,7 +293,7 @@ multi_line_output=3
 include_trailing_comma=True
 use_parentheses=True
 line_length=79
-known_compat=statsmodels.compat.*,setuptools,setuptools.*
+known_compat=statsmodels.compat.*,setuptools<65.6.0,setuptools.*
```

Comment:
This can't be right.  This is isort information. 

If you need to pin setuptools, should be in pyproject.toml.

Reply:
My bad, you are right